src/sentry/integrations/vsts/integration.py

```diff
@@ -631,43 +636,46 @@ def setup(self) -> None:
 
 class AccountConfigView(PipelineView):
     def dispatch(self, request: HttpRequest, pipeline: Pipeline) -> HttpResponseBase:
-        account_id = request.POST.get("account")
-        if account_id is not None:
-            state_accounts: Sequence[Mapping[str, Any]] | None = pipeline.fetch_state(
-                key="accounts"
-            )
-            account = self.get_account_from_id(account_id, state_accounts or [])
-            if account is not None:
-                pipeline.bind_state("account", account)
-                return pipeline.next_step()
-
-        state: Mapping[str, Any] | None = pipeline.fetch_state(key="identity")
-        access_token = (state or {}).get("data", {}).get("access_token")
-        user = get_user_info(access_token)
-
-        accounts = self.get_accounts(access_token, user["uuid"])
-        logger.info(
```

Comment:
do security or anyone use this log?

Reply:
i'm logging it as `extra` when we return the HTML template when there are no accounts. also emitting a failure there
https://github.com/getsentry/sentry/pull/80789/files#diff-ea1561c3fe6fedfc28168b76c65f65681c9efc769d6adbb617d99afaa598e867R665